datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py

```diff
@@ -293,6 +293,11 @@ def metadata(check):
                             current_check, row['metric_name'], MAX_DESCRIPTION_LENGTH
                         )
                     )
+                if row['interval'] and not isinstance(row['interval'], int):
```

Comment:
```suggestion
                if row['interval'] and not row['interval'].isdigit():
```

This line doesn't work as expected in the CI because row['interval'] is a string

Reply:
Hum. We should also check that the number is strictly positive.

Reply:
`'-1'.isdigit()` returns False